MVP/defaults-build.yaml

```diff
@@ -0,0 +1,74 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
+    node: 'jslave-platform-rhel7' #[MVP] please replace this value with jenkins slave name/label of your own.
+    #concurrent: true
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: tag 
+        - string:
+            name: name
+        - string:
+            name: version
+        - string:
+            name: release
```

Comment:
Those info came in handy when we wanna rebuild a previous job (e.g. one that failed immediately because of QEOS/Beaker incident) with all original parameters.
I agree it's not MVP's major case but seems little harm/noise to keep them for now?